frontend/src/modules/organization/components/form/organization-form-identities.vue

```diff
@@ -119,101 +57,88 @@ const props = defineProps({
     type: Object,
     default: () => {},
   },
-  showHeader: {
-    type: Boolean,
-    default: true,
-  },
-});
-const model = computed({
-  get() {
-    return props.modelValue;
-  },
-  set(newModel) {
-    emit('update:modelValue', newModel);
-  },
 });
 
-const identitiesForm = reactive({
+// TODO: move this to identities config
+const identitiesForm = {
   github: {
-    label: 'GitHub',
-    enabled:
-      props.modelValue.identities?.some((el) => el.platform === 'github')
-      || false,
     urlPrefix: 'github.com/',
   },
   linkedin: {
-    label: 'LinkedIn',
-    enabled:
-      props.modelValue.identities?.some((el) => el.platform === 'linkedin')
-      || false,
     urlPrefix: 'linkedin.com/company/',
   },
   twitter: {
-    label: 'X/Twitter',
-    enabled:
-      props.modelValue.identities?.some((el) => el.platform === 'twitter')
-      || false,
     urlPrefix: 'twitter.com/',
   },
   crunchbase: {
-    label: 'Crunchbase',
-    enabled:
-      props.modelValue.identities?.some((el) => el.platform === 'crunchbase')
-      || false,
     urlPrefix: 'crunchbase.com/organization/',
   },
-});
+};
 
-function findPlatform(platform) {
-  return CrowdIntegrations.getConfig(platform);
-}
+const model = ref([]);
 
-function onInputChange(newValue, key, value, index) {
-  model.value.identities[index] = {
-    ...props.modelValue.identities[index],
-    name: newValue,
-    url: newValue.length ? `https://${value.urlPrefix}${newValue}` : null,
-  };
-}
+watch(
+  props.modelValue,
+  (organization, previous) => {
+    if (!previous) {
+      const identities = organization.identities.map((i) => ({
+        ...i,
+        username: i.url ? i.url.split('/').at(-1) : null,
+      }));
+      const platforms = [...new Set(organization.identities.map((i) => i.platform))];
+      const noIdentity = Object.keys(identitiesForm)
+        .filter((platform) => !platforms.includes(platform))
+        .map((platform) => (reactive({
+          name: '',
+          platform,
+          url: null,
+          username: '',
+        })));
+      model.value = [
+        ...identities,
+        ...noIdentity,
+      ];
+    }
+  },
+  { deep: true, immediate: true },
+);
 
-function platformInIdentities(platform) {
-  return props.modelValue.identities.filter((i) => i.platform === platform);
-}
+watch(
+  model,
+  (value) => {
+    // Parse username object
+    const identities = value
+      .filter((i) => !!i.username?.trim() || !!i.name?.trim())
+      .map((i) => ({
+        ...i,
+        name: i.username,
+        url: i.username?.length ? `https://${identitiesForm[i.platform]?.urlPrefix}${i.username}` : null,
+      }));
```

Comment:
This logic, which is filtering identities that don't have a username is deleting identities such as `email`. Check this org https://app.crowd.dev/organizations/78863f71-5d4c-4df5-87b6-01261bf72356

Probably the same is happening with members